src/VisualStudio/Core/Def/FindReferences/Contexts/WithoutReferencesFindUsagesContext.cs

```diff
@@ -131,18 +131,27 @@ protected override async ValueTask OnDefinitionFoundWorkerAsync(DefinitionItem d
                 RoslynDefinitionBucket definitionBucket, DefinitionItem definition, CancellationToken cancellationToken)
```

Comment:
This is used to create `DefinitionItemEntry` for a definition with a single location above. If we have more then one source/metadata we call `TryCreateDocumentSpanEntryAsync` that creates `DocumentSpanItems`, which provide custom navigation.

If an item doesn't provide custom navigation VS uses the path for navigation and requires that the path exists on disk. 
Since both `DocumentSpanItems` and `DefinitionItemEntry` may represent symbols whose source doesn't exist on disk, they should both provide custom navigation.